src/__tests__/FileUpload.test.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import FileUpload, { validateExtension } from '../components/grader/FileUpload';
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+
+
+describe('Testing File input', () => {
```

Comment:
can you please run a formatter over all the project files? ensuring that it removes any extra white space (2+ empty lines) and also that every file ends with a new line